### PR TITLE
ref #299: article search index: styling

### PR DESCRIPTION
**Pull request by ckleis**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no  
| Deprecations? | no
| Fixed issues  | chameleon-system/chameleon-system#299
| License       | MIT

article search index: styling

### CODE REVIEW

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php [hunk 1]
```diff
@@ -1,30 +1,34 @@
-<div class="CMSInterfacePopup">
-    <form name="indexer" method="post" action="<?=PATH_CMS_CONTROLLER; ?>" accept-charset="UTF-8">
-        <input type="hidden" name="pagedef" value="<?=TGlobal::OutHTML($data['pagedef']); ?>"/>
-        <input type="hidden" name="_pagedefType" value="<?= $data['_pagedefType']; ?>"/>
-        <input type="hidden" name="module_fnc[<?=TGlobal::OutHTML($data['sModuleSpotName']); ?>]"
-               value="TickerIndexGeneration"/>
+<div class="card CMSInterfacePopup">
+    <div class="card-header">
+        Article search index
```
Comment (umulmrum) [inline src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: Could you add this text to translations and use the translated strings? Also please name it "Product search index" as we try to use the term "product" consistently.

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php [hunk 2]
```diff
@@ -1,30 +1,34 @@
-<div class="CMSInterfacePopup">
-    <form name="indexer" method="post" action="<?=PATH_CMS_CONTROLLER; ?>" accept-charset="UTF-8">
-        <input type="hidden" name="pagedef" value="<?=TGlobal::OutHTML($data['pagedef']); ?>"/>
-        <input type="hidden" name="_pagedefType" value="<?= $data['_pagedefType']; ?>"/>
-        <input type="hidden" name="module_fnc[<?=TGlobal::OutHTML($data['sModuleSpotName']); ?>]"
-               value="TickerIndexGeneration"/>
+<div class="card CMSInterfacePopup">
```
Comment (umulmrum) [inline src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]:
```suggestion
<div class="card">
```

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php [hunk 3]
```diff
@@ -1,30 +1,34 @@
...
+            <input type="hidden" name="module_fnc[<?=TGlobal::OutHTML($data['sModuleSpotName']); ?>]"
+                   value="TickerIndexGeneration"/>
 
-
-        <?php if (!$data['bIndexIsRunning'] || $data['bIndexCompleted']) {
-    ?>
-        <input type="submit" value="generate"/>
+            <?php if (!$data['bIndexIsRunning'] || $data['bIndexCompleted']) {
+                ?>
+                <input class="btn btn-primary" type="submit" value="generate"/>
```
Comment (umulmrum) [inline src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: Could you also add this text to translations (and with a capital "G")?

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php (hunk 1 above)
Comment (ckleis) [reply src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: OK, done.

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php (hunk 2 above)
Comment (ckleis) [reply src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: Done

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php (hunk 3 above)
Comment (ckleis) [reply src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: Done.

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php [hunk 4]
```diff
@@ -1,30 +1,44 @@
...
-        function SubmitCMSIndex() {
-            document.indexer.submit();
-        }
-        setTimeout("SubmitCMSIndex()", 1000);
-    </script>
-    <?php
-    } ?>
-</div>
\ No newline at end of file
+        <div class="mt-4"><?= $translator->trans('chameleon_system_shop.product_search_index.completed'); ?>: <?=$oLocal->FormatNumber($data['bPercentDone'], 2); ?>%</div>
```
Comment (umulmrum) [inline src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]:
```suggestion
        <div class="mt-4"><?= TGlobal::OutHTML($translator->trans('chameleon_system_shop.product_search_index.completed')); ?>: <?= TGlobal::OutHTML($oLocal->FormatNumber($data['bPercentDone'], 2)); ?>%</div>
```

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php [hunk 5]
```diff
@@ -1,30 +1,44 @@
...
+        <form name="indexer" method="post" action="<?=PATH_CMS_CONTROLLER; ?>" accept-charset="UTF-8">
+            <input type="hidden" name="pagedef" value="<?=TGlobal::OutHTML($data['pagedef']); ?>"/>
+            <input type="hidden" name="_pagedefType" value="<?= $data['_pagedefType']; ?>"/>
+            <input type="hidden" name="module_fnc[<?=TGlobal::OutHTML($data['sModuleSpotName']); ?>]"
+                   value="TickerIndexGeneration"/>
 
-    <?php if (!$data['bIndexCompleted'] && $data['bIndexIsRunning']) {
+            <?php if (!$data['bIndexIsRunning'] || $data['bIndexCompleted']) {
+                ?>
+                <input class="btn btn-primary" type="submit" value="<?= $translator->trans('chameleon_system_shop.product_search_index.generate'); ?>"/>
```
Comment (umulmrum) [inline src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]:
```suggestion
                <input class="btn btn-primary" type="submit" value="<?= TGlobal::OutHTML($translator->trans('chameleon_system_shop.product_search_index.generate')); ?>"/>
```

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php [hunk 6]
```diff
@@ -1,30 +1,44 @@
...
-        <?php
-} ?>
-    </form>
-    <?php
-    $oLocal = &TCMSLocal::GetActive();
-    ?>
-    Completed: <?=$oLocal->FormatNumber($data['bPercentDone'], 2); ?>%<br/>
+<div class="card">
+    <div class="card-header">
+        <?= $translator->trans('chameleon_system_shop.product_search_index.headline'); ?>
```
Comment (umulmrum) [inline src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]:
```suggestion
        <?= TGlobal::OutHTML($translator->trans('chameleon_system_shop.product_search_index.headline')); ?>
```

File: src/ShopBundle/Resources/translations/admin.de.xliff [hunk 7]
```diff
@@ -265,6 +265,18 @@
                 <source>chameleon_system_shop.list_manager_article.filter_active_all</source>
                 <target>Alle</target>
             </trans-unit>
+            <trans-unit id="chameleon_system_shop.product_search_index.headline">
+                <source>chameleon_system_shop.product_search_index.headline</source>
+                <target>Produktsuchindex generieren</target>
+            </trans-unit>
+            <trans-unit id="chameleon_system_shop.product_search_index.generate">
+                <source>chameleon_system_shop.product_search_index.generate</source>
+                <target>Generiere</target>
```
[the remark this answers is not in the document]
Comment (ckleis) [reply src/ShopBundle/Resources/translations/admin.de.xliff]: done

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php (hunk 4 above)
Comment (ckleis) [reply src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: ok, done

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php (hunk 5 above)
Comment (ckleis) [reply src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: done

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php (hunk 6 above)
Comment (ckleis) [reply src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: done

File: src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php (hunk 5 above)
Comment (ckleis) [reply src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php]: done

**Comment by umulmrum** [comment]
Thanks @ckleis

**Pull request merged**